src/js/component/reportForm/index.js

```diff
@@ -0,0 +1,168 @@
+import React, { useState } from 'react';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormControl from '@mui/material/FormControl';
+import FormLabel from '@mui/material/FormLabel';
+import Button from '@mui/material/Button';
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import Select from '@mui/material/Select';
+import DownloadIcon from '@mui/icons-material/Download';
+import { withStyles } from '@mui/styles';
+import styles from './styles';
+import { post } from '../../axiosUtils/appUtils';
+import { useDispatch } from 'react-redux';
+import { manageToast, setOverlayStatus } from '../../store/common/actions';
+
+const ReportForm = (props) => {
+  const { classes } = props;
+  const dispatch = useDispatch();
+  const [reportFormat, setReportFormat] = useState('daily');
+  const [selectedDate, setSelectedDate] = useState(null);
+  const [selectedMonth, setSelectedMonth] = useState('');
+  const [selectedYear, setSelectedYear] = useState('');
+
+  const months = [
```

Comment:
better to put all those in one js file and import it in this component. Tomorrow you might need them in other component too